terraform-zerotrust-postgrest-worker/gcp-instance-startup-script.tpl

```diff
@@ -74,14 +78,16 @@ ingress:
     service: ssh://localhost:22
   - service: http_status:404
 EOF
+
+# The credentials file does not get copied over so we'll do that manually 
+mkdir /etc/cloudflared || echo /etc/cloudflared already exists
+yes | sudo cp -via ~/.cloudflared/cert.json /etc/cloudflared/
```

Comment:
Nit:  you call it `cert`, as if it was a certificate file, when it is not;  that can raise confusion.
Maybe call it `creds.json`

Reply:
nice catch, copy-paste mistake :) fixed!